qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg

```diff
@@ -73,6 +73,8 @@
             balloon = balloon0
             balloon_dev_devid = balloon0
             balloon_dev_add_bus = yes
+            guest_compare_threshold = 300
```

Comment:
Could you give some introduction about why we need a threshold?Why we do not need it before?

Reply:
It's balloon specific config, you can check balloon_check related cases. If there's no threshold set, this case may fail sometimes(windows os' memory is not as controlled as Linux os).

Reply:
Okay,please provide test results here.

Reply:
> (1/1) Host_RHEL.m9.u3.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.win_virtio_driver_install_by_installer.driver_install.single_driver.with_balloon.q35: STARTED (1/1) Host_RHEL.m9.u3.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.win_virtio_driver_install_by_installer.driver_install.single_driver.with_balloon.q35: PASS (633.29 s) @leidwang Here are the latest results,thx

Okay, LGTM.

Just have a small question about the value of `guest_compare_threshold`, I see in some cases, the value is 150,and in some other cases, the value is 300. Could you explain why? What is its impact? Thanks！

Reply:
As you can searched, only balloon_minimum test case is set to 150, most test cases are set to 300, I thought  the previous one also should be set to 300, and the proper guest_compare_threshold value is talked for a long time, if you're interested you can search in internal bzs, there are several bugs talking about it.